projects/melon-runtime/Melon.Library/Data/data.js

```diff
@@ -0,0 +1,170 @@
+﻿const data = {
+    clone: function (target) {
+        if (typeof target === "function") {
+            return function () { target(...arguments ?? ''); }
+        }
+
+        return JSON.parse(JSON.stringify(eval(target)));
```

Comment:
I see the `eval` there :) Why is it necessary? Isn't target possibly a value that can be stringified?

Reply:
dfd1c923e08dcc90ab2b85a8c22de3be030db79d